### PR TITLE
Added 2 more CSS Questions

### PR DESCRIPTION
**Pull request by harsh1x4**
- made the right adjustment to the question number.
- Added the reference for question number 131.
- Created a CodePen for Question no. 133 and added it in the reference
- Added Question no. 134 but didn't knew the answer.

Co-authored-by: Alisha Takkar <alishatakkarconnect@gmail.com> 
Co-authored-by: mayank singh tomar <tomarm698@gmail.com>

### CODE REVIEW

File: css/css-quiz.md [hunk 1]
```diff
@@ -2088,3 +2090,36 @@ img {
...
+- [ ] horizontally centered
+- [ ] vertically centered, spanning the container's width
+- [ ] at the bottom of the container, spanning the container's width
+- [x] at the top of the container, spanning the container's width
+
+[Reference](https://codepen.io/harsh1x4/pen/poVaQVz)
+
+#### Q134. By default, a CSS transition is __
+
+- [ ] asymmetric, so that the transition going from the initial state to the end state is different from the transition going from the end state back to the initial state.
```
Comment (201B153) [inline css/css-quiz.md:2120]: The answer for Q134. will be (C)
u can refer [!reference](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions)
Please update it and PR again.

**Review by 201B153 — CHANGES_REQUESTED** [review]

File: css/css-quiz.md (hunk 1 above)
Comment (harsh1x4) [reply css/css-quiz.md:2120]: Thank You.✌

**Review by 201B153 — APPROVED** [review]
Grt looks good

**Review by Ebazhanov — APPROVED** [review]
Thank you @harsh1x4 for your valuable contribution! Great job! 👍

**Comment by Ebazhanov** [comment]
@all-contributors please add @harsh1x4 for design, code

**Pull request merged**